paper/paper.md

```diff
@@ -0,0 +1,126 @@
+  with other software super-easy.
+
+* **Richer support with sister packages**
+
+  pyOBIS when coupled with sister packages e.g. `pyDwcViz` can be utilized to perform many important computations easily.
+  With one-line function and plug-and-play use,
+  users can generate biodiversity indices such as `ES50` and `Shannon's Index`,
+  get environment statistics from occurrence records queried for specified geo-spatial region of interest,
+  taxa, or other paramters,
+  generate interactive distribution plots with taxanomic heirarchy easily,
+  and many other possible use cases.
+
+# Figures
+![Absolute Depth for Lepidochelys kempii over time.\label{fig:time-series-turtle}](https://github.com/ayushanand18/pyobis/assets/36472216/b6e66f31-7bbd-49c9-8186-3ab1a58e57c0)
+
+pyOBIS can be used to do super-useful time series analysis for instance, absolute depth of Sea Turtle species, Lepidochelys kempii between 1990-2011 as shown in figure \autoref{fig:time-series-turtle}. From this analysis, the following observations can be made:
+* The average depth has increased over the years, this means the species is looking for cooler waters to escape the heating waters. (This can be observed from the magenta-colored line which depicts the 5-year rolling average.)
+* The species has witnessed a slight compression, i.e., minimal and maximal depth have come closer. For a brief period, it compressed significantly (around 2006) this might be due to data constraints or maybe some seasonal current. After that it has regained a lot but still the average difference in minimal and maximal depth is lower than early 2000s.
+* However, necessary precautions to avoid sampling bias must be taken into consideration.
+
+# Conclusion
+The pyOBIS python package provides a convenient and efficient way to access and work with taxonomic occurrence records from the Ocean Biodiversity Information System (OBIS).
+With over 107 million occurrence records,
+OBIS is a valuable resource for marine biodiversity data,
+but its accessibility has been a challenge which pyOBIS addresses by offering built-in functions for retrieving data on occurrences, taxons, nodes, checklists, and dataset metadata.
+It enables researchers to download,
+visualize, segment, process, and export data in various formats using its tools or other Python libraries.
+By integrating with sister packages like pyDwcViz,
+pyOBIS enhances its capabilities for analyzing Darwin Core Data with ease.
+Overall, pyOBIS simplifies the handling of OBIS data,
+facilitates data exploration and analysis,
+and empowers researchers to study global and local species distributions across space and time.
+
+# Acknowledgements
+We acknowledge the help of `Pandas`, `Matplotlib`, and `requests` python package, and all the authors for their contributions building this package, performing the associated analysis and drafting this manuscripts.
+
+# References
```

Comment:
We are not putting any citations etc. Should we remove this heading entirely?

Reply:
We should probably mention some OBIS "canonical" reference and/or mention similar efforts on the R-land and mentioned them.